qiskit/compiler/transpile.py

```diff
@@ -238,45 +288,38 @@ def _parse_transpile_args(circuits, backend,
     arg has more priority than the arg set by backend).
 
     Returns:
-        list[TranspileConfig]: a transpile config for each circuit, which is a standardized
-            object that configures the transpiler and determines the pass manager to use.
+        list[dicts]: a list of transpile parameters.
     """
     # Each arg could be single or a list. If list, it must be the same size as
     # number of circuits. If single, duplicate to create a list of that size.
     num_circuits = len(circuits)
 
     basis_gates = _parse_basis_gates(basis_gates, backend, circuits)
-
-    coupling_map = _parse_coupling_map(coupling_map, backend, num_circuits)
-
+    coupling_map = _parse_coupling_map(coupling_map, backend, circuits)
     backend_properties = _parse_backend_properties(backend_properties, backend, num_circuits)
-
     initial_layout = _parse_initial_layout(initial_layout, circuits)
-
     seed_transpiler = _parse_seed_transpiler(seed_transpiler, num_circuits)
-
     optimization_level = _parse_optimization_level(optimization_level, num_circuits)
-
     pass_manager = _parse_pass_manager(pass_manager, num_circuits)
+    output_name = _parse_output_name(output_name, num_circuits)
 
-    output_name = _parse_output_name(output_name, circuits)
-
-    transpile_configs = []
+    transpile_args_circuits = []
```

Comment:
suggest better name, as there are no circuits involved in the dict
```suggestion
    list_transpile_args = []
```

Reply:
Sure... fixed in 4c87790db